Neo4j.Driver/Neo4j.Driver.Tck.Tests/TCK/DriverResultApi.feature

```diff
@@ -86,7 +86,7 @@ Feature: Tests the uniform API of the driver
     Then requesting the `Statement` as text should give: CREATE (:label {name: "Pelle"} )
     And requesting the `Statement` parameter should give: {}
 
-  Scenario: Access Statement parametrised
+  Scenario: Access Statement parameterised
```

Comment:
Please ignore any spelling errors in path `Neo4j.Driver/Neo4j.Driver.Tck.Tests/TCK`
They are auto generated.